packages/plugins/experimental/plugin-calls/src/capabilities/react-context.tsx

```diff
@@ -6,22 +6,18 @@ import React from 'react';
 
 import { Capabilities, contributes } from '@dxos/app-framework';
 
-import { CallManager } from '../call';
-import { CallsGlobalContext } from '../hooks';
+import { CallGlobalContextProvider } from '../components/CallGlobalContextProvider';
 import { CALLS_PLUGIN } from '../meta';
 
 export type CallsPluginOptions = {};
 
 export default (_options: CallsPluginOptions = {}) => {
-  // Create a global live object containing the call state.
-  const call = new CallManager();
-
   return contributes(
     Capabilities.ReactContext,
     {
       id: CALLS_PLUGIN,
       context: ({ children }) => {
-        return <CallsGlobalContext.Provider value={{ call }}>{children}</CallsGlobalContext.Provider>;
+        return <CallGlobalContextProvider>{children}</CallGlobalContextProvider>;
```

Comment:
does this make calls context inaccesbile outside react? For example from graph builder

Reply:
Do not know the answer to that, but probably yes, and we need to move it to capability. cc @wittjosiah

Reply:
correct, yes we need to move to capability to make it accessible outside react